### ISSUE TITLE
Help rendered twice

### ISSUE THREAD

**Issue by Shatur** [ibhagwan/fzf-lua#415]
### Info

- Operating System: ArchLinux
- Shell: bash
- Terminal: Alacritty
- `nvim --version`: 0.7
- `fzf --version`: 0.30.0 (2093667)

### Description 

`:Fzf help_tags` renders twice:

![изображение](https://user-images.githubusercontent.com/22453358/168446895-1a82442a-fd25-4f2c-8a6d-e877b282837a.png)

Is this expected?

P.S: Sorry for a lot of opened issues, just migrating from Telescope :)

**Comment by ibhagwan** [comment]
This is actually by design, instead of saving all the help tags and their corresponding help files in memory (like Telescope) FzfLua uses Neovim’s builtin `:help` command to locate the help tag and then renders the help buffer inside the preview window.

Unfortunately, you cannot open `:help` in a cusrom/hidden buffer, neovim always opens a single help buffer, what FzfLua does is open the help buffer and then “minimize” it to a single line at the bottom and then display the same help buffer contents inside the preview win.

What I suspect is happening in your case (which is btw, exactly how my setup works) is that you have an `autocmd` for help buffers to open at certain size/location (I.e. `:wincmd L`) which then overrides the “minimization” of the help buffer by FzfLua.

TL;DR, this is harmless and “by design” :)


> P.S: Sorry for a lot of opened issues, just migrating from Telescope :)

Dw about it, this is the only way this plugin becomes perfect (arguably it’s almost there). Mind me asking why are you moving from Telescope? Performance?

**Comment by Shatur** [comment]
Got it, thank you!

> Performance?

Yes :( It's very slow on big repos.

**Comment by Shatur** [comment]
Added `if vim.api.nvim_win_get_height(0) ~= 1 then` check to my config.

**Comment by ibhagwan** [comment]
> Added `if vim.api.nvim_win_get_height(0) ~= 1 then` check to my autocmd.

That’s a great idea! I’ll add it to my setup :)
